chart/templates/scheduler/scheduler-deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
 {{- if $stateful }}
   serviceName: {{ .Release.Name }}-scheduler
 {{- end }}
-  replicas: 1
+  replicas: {{ .Values.scheduler.replicas }}
```

Comment:
I wonder if there's any way we can guard this to only work for 2.0.0 -- but given users might use their own Docker image I don't think there's much we can really do.

I guess just make it clear in the values.yaml file do not set this >1 unless you are on 2.0.0.

Reply:
@ashb yeah I don't think there's a warning, maybe we just need to be clear in documentation :/.